### DISCUSSION TITLE
Revoke session of user

### DISCUSSION THREAD

**Discussion by janiskemper** [ory/kratos#924] [Q&A] [answered]
Is there a possibility to revoke a user's session without having the actual cookie and performing the standard logout flow? Use case would be if the phone/laptop is stolen and the session should be revoked so that the thief cannot cause any harm with it.

**Comment by zepatrik** [comment, marked as answer]
Not yet but it is definitely on our list, see https://github.com/ory/kratos/issues/655

**Reply by janiskemper** [reply]
Okay, thank you!
